core: add user switching (#22659)
goauthentik/authentik: website/docs/add-secure-apps/flows-stages/stages/account_selection/index.md

---

> title: Account selection stages

Reviewer: ```suggestion
title: Account selection stage
```

---

> Account selection stages let users choose which remembered authentik account should continue a flow.

Reviewer: ```suggestion
The account selection stage allows users choose which remembered authentik account should continue a flow.
```

## Overview

> authentik can remember multiple accounts in the same browser after successful login. The remembered account list is stored in a signed browser cookie, but an account is only shown when its authentik session is still active.

Reviewer: ```suggestion
authentik can remember multiple accounts within the same browser. The list of remembered accounts is stored in a signed browser cookie, and an account only appears if its authentik session is still active.
```

> Two stages work together:

Reviewer: I think this would need to be split in to 2 docs. Maybe within a directory in the sidebar with an overview page?
